src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -82,6 +84,9 @@ public static ICapabilityStatementBuilder Create(IModelInfoProvider modelInfoPro
 
             statement.FhirVersion = modelInfoProvider.SupportedVersion.VersionString;
             statement.Date = ProductVersionInfo.CreationTime.ToString("O");
+
+            Uri baseUri = new Uri(urlResolver.ResolveMetadataUrl(false).ToString());
+            statement.Url = new Uri(baseUri, "metadata");
```

Comment:
The resolver looks like it should already add /metadata, any thoughts on why this isn't working?

https://github.com/microsoft/fhir-server/blob/05f5275f4275893c5a526cb1538be05daf1580a0/src/Microsoft.Health.Fhir.Api/Features/Routing/UrlResolver.cs#L99-L103

Reply:
You are right, here is the fix https://github.com/microsoft/fhir-server/pull/3274 if you can check it